src/Domain/Item/ItemService.ts

```diff
@@ -186,7 +203,17 @@ export class ItemService implements ItemServiceInterface {
     const ourUpdatedAtTimestamp = existingItem.updatedAt
     const difference = incomingUpdatedAtTimestamp - ourUpdatedAtTimestamp
 
-    return Math.abs(difference) > this.MIN_CONFLICT_INTERVAL_MICROSECONDS
+    return Math.abs(difference) > this.getMinimalConflictIntervalMicroseconds(apiVersion)
+  }
+
+  private getMinimalConflictIntervalMicroseconds(apiVersion?: string): number {
```

Comment:
I would honestly think it fine to return `Time.MicrosecondsInAMillisecond` for all versions, but this is also fine if you want to go the safer route.

However, if anything, it seems you should return `Time.MicrosecondsInASecond` for the older API versions, and default: should be `Time.MicrosecondsInAMillisecond`.

Reply:
Will fix 👍